core/identity/src/dao/identity.rs

```diff
@@ -77,6 +77,11 @@ impl<'c> IdentityDao<'c> {
                 diesel::insert_into(s::identity::table)
                     .values(&preconfigured_identity)
                     .execute(conn)?;
+                diesel::update(s::identity::table)
+                    .set(id_dsl::is_default.eq(false))
```

Comment:
This is not currently the case, although `preconfigured_identity.is_default` equal to `false` is legal. Since we're already assuming that this will be the default identity, we can enforce `preconfigured_identity.is_default = true;`

Reply:
I don't understand this comment